goals.md

```diff
@@ -9,29 +9,29 @@ Based on this, we would like to bring the goals to the TODO Group for review and
 
 The goals will be revisited and updated yearly.
 
-## 2022
+## [PROPOSED] 2022
 
-### 1. Cultivate growth and adoption of OSPOs
+### 1 Cultivate growth and adoption of OSPOs
 
-* Release at least 6 new OSPO case studies
-* Launch 6 new OSPO guides
-* Host 12 OSPOlogy sessions
-* Build out and run OSPO Survey 2022
-* Create and publish a OSPO Maturity Model
+- Release at least 6 new OSPO case studies
+- Launch 6 new OSPO guides
+- Host 12 OSPOlogy sessions
+- Build out and run OSPO Survey 2022
+- Create and publish a OSPO Maturity Model
```

Comment:
@vmbrasseur I think the GGI from OW2 is a very useful resource for the open source community, but my impression is that the message and audience are more for individuals and organizations starting and advancing in open source and not necessarily running an OSPO in the organization.

On the other hand, there's some context that should be added to this topic: This idea comes from a bigger issue coming from the community that is the lack of structure, and context when looking at the wide diversity of TODO resources. (TODO Resources are super useful, but a recurrent feedback I get at meetings, surveys, or email threads is that there is no order nor a defined path for the OSPO Community to understand where, when, and how to use those resources) + There is a WIP OSPO research study with involvement from a bunch of OSPOs that has been under development since June - July AFAIK. Maybe @caniszczyk could add more details.